src/main/utility/timing.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TIMING_H_
+#define TIMING_H_
+#include <stdio.h>
+#include <time.h>
+
+#define RECORD_TIME(TS_NAME) \
+    struct timespec TS_NAME; \
+    clock_gettime(CLOCK_REALTIME, &TS_NAME);
```

Comment:
I think we want `CLOCK_MONOTONIC` here instead of `CLOCK_REALTIME` because `CLOCK_REALTIME` can jump around if admin users on the machine adjust the clock between our start and stop stamps.

Reply:
@robgjansen I think this file and the calls to `RECORD_TIME` were left over from when I was trying to get a tight measure on `fork` performance. I think we can just remove this code.